pkg/networkutils/network_test.go

```diff
@@ -363,6 +363,10 @@ func TestSetupHostNetworkNodePortEnabledAndSNATDisabled(t *testing.T) {
 				},
 			},
 		},
+		"nat": {
+			"AWS-SNAT-CHAIN-0":     [][]string{{"AWS-SNAT-CHAIN-0"}},
```

Comment:
Is this first entry with the chain name supposed to represent the `-N` entry?

Reply:
yes

Reply:
It seems a bit weird to pass the chain name twice, as it is hard to tell that the name as the first entry maps to a `-N` entry. I think we should either pass the string `-N`, or infer from the chain